robot/camera.py

```diff
@@ -6,6 +6,49 @@
 from robot.markers import Marker
 
 
+class ResultList(MutableSequence):
```

Comment:
Why does this definition need to move?

Reply:
We could also take the opportunity to un-break this class since it currently won't typecheck properly anyway.